server/verify.go

```diff
@@ -318,10 +322,10 @@ func makePool(certs []*x509.Certificate) *x509.CertPool {
 	return pool
 }
 
-// VerifyGceTechnology checks the GCE-specific GceNonHost event's Trusted Execution Technology (TEE)
```

Comment:
If we unexport this, it's a backwards incompatible change. Please add a breaking comment to the PR description.

Reply:
Done.